src/explorer/documentListTreeItem.ts

```diff
@@ -17,11 +17,14 @@ const log = createLogger('documents tree item');
 export const MAX_DOCUMENTS_VISIBLE = 10;
 
 export const DOCUMENT_LIST_ITEM = 'documentListTreeItem';
-export enum CollectionTypes {
-  collection = 'collection',
-  view = 'view',
-  timeseries = 'timeseries',
-}
+export const COLLECTION_TYPES = {
```

Comment:
same idea, keep the name and type the same. I won't comment on the rest we can just discuss on my `DocumentSource` comment